flowmachine/flowmachine/core/server/action_handlers.py

```
@@ -93,6 +110,11 @@ def _load_query_object(params: dict) -> "BaseExposedQuery":
             params,
             orig_error_msg=orig_error_msg,
         )
+        raise QueryLoadError(
+            error_msg,
+            params,
+            orig_error_msg=orig_error_msg,
+        )
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)